src/pages/Home.jsx

```diff
@@ -28,25 +29,49 @@ const Home = () => {
   const isReallySmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
   const iconSize = isReallySmallScreen ? 'medium' : 'large';
 
+  // Logo section elements
+  const heading = 'PASS';
+  const punctuation = (
+    <Box component="span" sx={visuallyHidden}>
+      :
+    </Box>
+  );
+  const subheading = 'Personal Access System for Services';
+
   const logoSection = isReallySmallScreen ? (
-    <Stack justifyContent="center" alignItems="center" spacing={2} mb={2}>
-      <Typography variant="h1" fontWeight="500" fontSize="72px" color="primary">
-        PASS
-      </Typography>
-      <Box component="img" src="/assets/PASSLogolightmode.png" alt="PASS logo" width="50%" />
-    </Stack>
+    <Typography component="h1" color="primary">
+      <Stack component="span" justifyContent="center" alignItems="center" spacing={2} mb={2}>
+        <Typography variant="h1" component="span" fontWeight="500" fontSize="72px">
+          {heading}
+          {punctuation}
+        </Typography>
+        <Box component="img" src="/assets/PASSLogolightmode.png" alt="" width="50%" />
+        <Typography variant="h4" component="span" fontWeight="600" mb={8}>
+          {subheading}
+        </Typography>
+      </Stack>
+    </Typography>
   ) : (
-    <Stack direction="row" justifyContent="center" alignItems="center" spacing={4}>
-      <Box component="img" src="/assets/PASSLogolightmode.png" alt="PASS logo" width="150px" />
-      <Typography
-        variant="h1"
-        fontWeight="500"
-        fontSize={isReallySmallScreen ? '72px' : '144px'}
-        color="primary"
-      >
-        PASS
-      </Typography>
-    </Stack>
+    <Typography component="h1" color="primary">
+      <Stack direction="column" justifyContent="center" alignItems="center" spacing={4}>
+        <Stack
+          component="span"
+          direction="row"
+          justifyContent="center"
+          alignItems="center"
+          spacing={4}
+        >
+          <Box component="img" src="/assets/PASSLogolightmode.png" alt="" width="150px" />
+          <Typography variant="h1" component="span" fontWeight="500" fontSize="144px">
+            {heading}
+            {punctuation}
+          </Typography>
+        </Stack>
+        <Typography variant="h3" component="span" fontWeight="600" mb={18}>
```

Comment:
Can we move this `mb={18}` from here (line 70) to the `<Stack>` on line 56 instead?  This will look as intended. 

I would approve with this change.

Reply:
pushed this fix